test/crlf.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+var test = require('tap').test
+var pino = require('../')
+var Writable = require('stream').Writable
+
+class LogWriteStream extends Writable {
```

Comment:
I realise it's just test, but we don't use classes - take a look at the other tests 
for how we do this (helpers.sink)